frontend/public/style/_overrides.scss

```diff
@@ -497,3 +497,8 @@ button.pf-c-dropdown__menu-item.pf-m-disabled {
 #modal-container .pf-c-backdrop {
   position: absolute !important;
 }
+
+.text-muted{
+  color: var(--pf-global--Color--200);
```

Comment:
We also want to apply this color to `co-status-card__health-item-subtitle`

Reply:
> We also want to apply this color to `co-status-card__health-item-subtitle`

@spadgett I have changed the color for `co-status-card__health-item-subtitle` to `var(--pf-global--Color--200)`, and left `text-muted` to inherit parent size. Updated screenshot is attached above. 
Please take a look if this looks good.